pkg/datapath/loader/xdp.go

```diff
@@ -19,13 +19,9 @@ import (
 
 func xdpModeToFlag(xdpMode string) uint32 {
 	switch xdpMode {
-	case option.XDPModeNative:
+	case option.XDPModeNative, option.XDPModeLinkDriver, option.XDPModeBestEffort:
```

Comment:
Hmm, now `XDPModeBestEffort` in a way implies `XDPModeLinkDriver`. Perhaps it'd be better to reflect that in the name? `XDPModeLinkDriverBestEffort`or `XDPModeTryLinkDriver` come to mind, but they're a bit of a mouthful.

Reply:
Happy to update this to whatever consensus the Cilium project can reach!

Reply:
For what it's worth I like `XDPModeTryLinkDriver`, it's not that long.

Reply:
Does anyone have a strong opinion on this?

Keep in mind that BestEffort is what was originally suggested by @borkmann so this wasn't just me pulling something out of thin air!

Reply:
@borkmann done